File: app/src/main/java/com/moufee/purduemenus/repository/data/menus/Favorites.java
package com.moufee.purduemenus.repository.data.menus;

import androidx.annotation.Keep;

import com.squareup.moshi.Json;

import java.util.List;

/**
 * A container for a list of favorites
 */
@Keep
public class Favorites {
    @Json(name = "Favorite")
    private List<Favorite> mFavorites;

    public List<Favorite> getFavorites() {
        return mFavorites;
    }
}

File: app/src/main/java/com/moufee/purduemenus/repository/data/menus/Favorites.kt
package com.moufee.purduemenus.repository.data.menus

import androidx.annotation.Keep
import com.squareup.moshi.Json

/**
 * A container for a list of favorites
 */
@Keep
class Favorites {
    @Json(name = "Favorite")
    val favorites: List<Favorite>? = null
}